### ISSUE TITLE
how l can disable events on Knob?

### ISSUE THREAD

**Issue by matador96** [eskimoblood/rc-knob#23]
Hello, how l can disable any events (like onClick or onKeyDown) on Knob?

**Comment by eskimoblood** [comment]
Mhmm, that's something I've overlooked yet. But in the end it's easy as all it needs is a disabled flag that will prevent the adding of the events to the outer `div`. Do you feel comfortable to open an PR for it or should I do it, but I'm not sure when I will find time for it.

**Comment by matador96** [comment]
> Mhmm, that's something I've overlooked yet. But in the end it's easy as all it needs is a disabled flag that will prevent the adding of the events to the outer `div`. Do you feel comfortable to open an PR for it or should I do it, but I'm not sure when I will find time for it. 

Yeah, l know that, but l think maybe have default functions or props for this.

Thanks for answer ;)
